pipelines/rj_cvl__osinfo_mongo/tasks.py

```
@@ -0,0 +1,240 @@
+        Each chunk has: {'n': int, 'data': bytes}
+    """
+    client = bigquery.Client()
+
+    # Build query with files_id filter
+    files_ids_str = ", ".join([f"'{fid}'" for fid in files_ids])
+    query = f"""
+        SELECT files_id, n, data
+        FROM `{client.project}.{dataset_id}.{table_id}`
+        WHERE files_id IN ({files_ids_str})
+        ORDER BY files_id, n
+    """
```

Ruff (0.14.8)

Possible SQL injection vector through string-based query construction

(S608)